src/components/badge.component.js

```diff
@@ -25,6 +25,7 @@ const BadgeContainer = styled.View`
 const BadgeText = styled.Text`
   ${{ ...fonts.fontPrimaryBold }};
   background-color: transparent;
+  ${({ color }) => `color: ${color || colors.black};`};
```

Comment:
It reminds me a question that, what's the differences between this way with `color: ${{color} => (color || color.black)};`?

Reply:
Both styles are OK. But the one I mentioned is more similar with styled-components [official documents](https://www.styled-components.com/docs/basics#passed-props).

Reply:
@chinesedfan just had the time to test, it's exactly the same.
I switched to the syntax you proposed as it's clearer.

I guess the difference between the two syntaxes, is that with the one I originally used, you can choose to output no style at all based on tests, which is not possible with the other one:

```js
${({ foo }) => foo ? `color: '#FFCC00'` : ''};
```

You could also output more styles in the condition